SignalShareExtension/ShareViewController.swift

```diff
@@ -322,6 +327,14 @@ public class ShareViewController: UINavigationController, SAELoadViewDelegate, S
         Logger.debug("\(self.logTag()) \(#function)")
 
         super.viewDidAppear(animated)
+
+        // We can't show the conversation picker until the DB is set up.
+        // Normally this will only take a moment, so rather than flickering and then hiding the loading screen
+        // We start with as invisible, and only fade it in if it's going to take a while
+        self.view.isHidden = false
+        UIView.animate(withDuration: 0.1, delay: 0.5, options: [.curveEaseInOut], animations: {
+            self.view.alpha = 1
+        }, completion: nil)
```

Comment:
I think you've got a race here, because `presentConversationPicker()`  could be called before or after this animation.  There's a bunch of possible solutions, but I suggest that you:

* Move this animation to loadView() to ensure the animation is always started first.
* `self.view.layer.removeAllAnimations()` everywhere we do `self.present(...)` to ensure that we cancel the animation if necessary when presenting our picker _or_ an error view.

Reply:
What is the harm of self.view controller becoming "visible" behind the conversationPicker?

Reply:
Ah, I guess it could be weird to see a blue screen fade in behind the conversation picker as the picker is being presented?

I've addressed that with another commit. PTAL @charlesmchen